packages/ssg-plugins-essential/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@hono/ssg-plugins-essential",
+  "version": "1.0.0",
```

Comment:
The version number will be increased by Changeset automatically. So, please specify it as `0.0.0`.